Demo1.js

```diff
@@ -27,6 +27,15 @@ export default React.createClass({
     };
   },
 
+  handleTouchStart: function(key, pressLocation, event) {
+    this.handleMouseDown(key, pressLocation, event.touches[0]);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/touches

Maybe to destructuring? Looking at what you were actually using in `MouseDown` and `MouseMove` I suspected that a TouchList touches object would have the same properties and I could get away with it.

It would be fun to try supporting multiple touches on this and seeing what happens :)